pallets/liquid-staking-v2/Cargo.toml

```diff
@@ -14,6 +14,8 @@ frame-system       = { git = 'https://github.com/paritytech/substrate.git', bran
 sp-runtime         = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.8', version = '3.0.0', default-features = false }
 sp-std             = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.8', version = '3.0.0', default-features = false }
 orml-traits        = { git = 'https://github.com/open-web3-stack/open-runtime-module-library.git', default-features = false }
+xcm                = { git = 'https://github.com/paritytech/polkadot.git', branch = 'release-v0.9.8', default-features = false }
+max-encoded-len    = { git = 'https://github.com/paritytech/substrate', branch = 'polkadot-v0.9.8', default-features = false }
```

Comment:
let's use codec's `max-encoded-len` feature

Reply:
Fix it in next commit.

Reply:
you can click resolve if it's solved